src/interfaces/ICSParametersRegistry.sol

```diff
@@ -0,0 +1,262 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+interface ICSParametersRegistry {
+    struct markedUint248 {
```

Comment:
wdyt if we make it a custom type?
it allows to hide cast to uint248 and "always true" boolean argument

Reply:
I think it is not worth the efforts